### ISSUE TITLE
Tests of `MemoryHelper`

### ISSUE THREAD

**Issue by Vixenka** [NoiseStudio/NoiseEngine#354]
https://github.com/NoiseStudio/NoiseEngine/blob/98d8bfa1105f6e720f064bc3667ee9548008cfc7/NoiseEngine/Common/MemoryHelper.cs#L15

https://github.com/NoiseStudio/NoiseEngine/blob/98d8bfa1105f6e720f064bc3667ee9548008cfc7/NoiseEngine/Common/MemoryHelper.cs#L25

**Comment by kkard2** [comment]
isn't this fixed by #361?

**Comment by Vixenka** [comment]
@kkard2 good point, but it was fixed partialy, because only `AligmentOf` have their tests.
